shell-completion/Makefile

```diff
@@ -40,15 +40,15 @@ all: hledger-completion.bash
 install:
 	@install -v -d "$(DESTDIR)$(BASHCOMPDIR)"
```

Comment:
Just a question, why are these command's echos silenced with `@`? Just for `install` and `uninstall` because of "no news is good news"?

Reply:
For more user-friendly output, they are invoked verbosely, so it makes echoing redundant.

Reply:
A, I see now.